src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatereasonableadjustment/UpdateReasonableAdjustmentAboutToSubmitHandler.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.updatereasonableadjustment;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isNoOrNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isYes;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+@Component
+@Slf4j
+public class UpdateReasonableAdjustmentAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
+        return callbackType == CallbackType.ABOUT_TO_SUBMIT
+            && callback.getEvent() == EventType.UPDATE_REASONABLE_ADJUSTMENT
+            && nonNull(callback.getCaseDetails())
+            && nonNull(callback.getCaseDetails().getCaseData());
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback, String userAuthorisation) {
+        if (!canHandle(callbackType, callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+
+        SscsCaseData sscsCaseData = callback.getCaseDetails().getCaseData();
+
+        if (isYes(sscsCaseData.getAppeal().getAppellant().getIsAppointee()) || nonNull(sscsCaseData.getReasonableAdjustment().getAppellant()) && isNoOrNull(sscsCaseData.getReasonableAdjustment().getAppellant().getWantsReasonableAdjustment())) {
```

Comment:
Is this logic correct? It's saying if there is an appointee on the case OR the reasonable adjustment for appellant is no/null. So this logic will hit whenever there is an appointee on the case?

Reply:
your explanation of the logic is correct. 

We do not send letters to appellants when there is an appointee on the appeal. 
In this case, you should not be able to set the alternative letter formats for the appellant since there is an appointee, for this reason the appellant's value is set to null.

Reply:
ok